src/java/picard/analysis/MeanQualityByCycle.java

```diff
@@ -52,9 +52,23 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Program to generate a data table and pdf chart of " +
-                "mean base quality by cycle from a SAM or BAM file.  Works best on a single lane/run of data, but can be applied to" +
-                "merged BAMs. Uses R to generate chart output.",
+        usage = "This metric gives an overall snapshot of sequencing machine performance, as base quality scores" +
```

Comment:
Start with stating what the tool does, not defining the metric. I know it should be obvious how they relate to each other, but for non-native speakers it really helps to have the same super-obvious pattern for each tool doc.